--- ./src/gpg-error-config.in.libdir	2010-01-14 10:18:30.000000000 +0100
+++ ./src/gpg-error-config.in	2010-08-15 17:50:02.000000000 +0200
@@ -13,7 +13,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
-libdir=@libdir@
 isubdirafter="@GPG_ERROR_CONFIG_ISUBDIRAFTER@"
 
 output=""
@@ -69,9 +68,6 @@
 	    output="$output @GPG_ERROR_CONFIG_CFLAGS@"
 	    ;;
 	--libs)
-	    if test "x$libdir" != "x/usr/lib" -a "x$libdir" != "x/lib"; then
-		output="$output -L$libdir"
-	    fi
 	    output="$output @GPG_ERROR_CONFIG_LIBS@"
 	    ;;
 	*)

