diff -Naur libgpg-error-1.12.orig/configure.ac libgpg-error-1.12/configure.ac
--- libgpg-error-1.12.orig/configure.ac	2013-12-24 07:07:30.110500334 -0500
+++ libgpg-error-1.12/configure.ac	2013-12-24 07:10:23.274502314 -0500
@@ -217,6 +217,7 @@
 AC_CONFIG_FILES([lang/Makefile lang/cl/Makefile lang/cl/gpg-error.asd])
 AC_CONFIG_FILES([src/versioninfo.rc])
 AC_CONFIG_FILES([src/gpg-error-config], [chmod +x src/gpg-error-config])
+AC_CONFIG_FILES([src/gpg-error.pc])
 
 AC_OUTPUT
 
diff -Naur libgpg-error-1.12.orig/src/gpg-error.pc.in libgpg-error-1.12/src/gpg-error.pc.in
--- libgpg-error-1.12.orig/src/gpg-error.pc.in	1969-12-31 19:00:00.000000000 -0500
+++ libgpg-error-1.12/src/gpg-error.pc.in	2013-12-24 07:10:23.274502314 -0500
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: gpg-error
+Description: a library that defines common error values for all GnuPG components
+Version: @VERSION@
+Libs: -L{libdir} -lgpg-error
+Cflags: -I${includedir}
diff -Naur libgpg-error-1.12.orig/src/Makefile.am libgpg-error-1.12/src/Makefile.am
--- libgpg-error-1.12.orig/src/Makefile.am	2013-12-24 07:07:30.108500334 -0500
+++ libgpg-error-1.12/src/Makefile.am	2013-12-24 07:10:23.274502314 -0500
@@ -37,13 +37,15 @@
 bin_SCRIPTS = gpg-error-config
 m4datadir = $(datadir)/aclocal
 m4data_DATA = gpg-error.m4
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = gpg-error.pc
 
 EXTRA_DIST = mkstrtable.awk err-sources.h.in err-codes.h.in \
 	mkerrnos.awk errnos.in README \
 	mkerrcodes.awk mkerrcodes1.awk mkerrcodes2.awk mkerrcodes.c \
 	mkheader.awk gpg-error.h.in mkw32errmap.c w32-add.h w32ce-add.h \
 	err-sources.h err-codes.h gpg-error-config.in gpg-error.m4 \
-	gpg-error.def.in versioninfo.rc.in
+	gpg-error.def.in versioninfo.rc.in gpg-error.pc.in
 
 BUILT_SOURCES = err-sources.h err-codes.h code-to-errno.h code-from-errno.h \
 	err-sources-sym.h err-codes-sym.h errnos-sym.h gpg-error.h \
