diff -Naur libgpg-error-1.12.orig/src/gpg-error-config.in libgpg-error-1.12/src/gpg-error-config.in
--- libgpg-error-1.12.orig/src/gpg-error-config.in	2013-12-24 07:07:30.108500334 -0500
+++ libgpg-error-1.12/src/gpg-error-config.in	2013-12-24 07:09:05.360501423 -0500
@@ -13,7 +13,6 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
-libdir=@libdir@
 isubdirafter="@GPG_ERROR_CONFIG_ISUBDIRAFTER@"
 
 output=""
@@ -69,12 +68,6 @@
 	    output="$output @GPG_ERROR_CONFIG_CFLAGS@"
 	    ;;
 	--libs)
-	    case "$libdir" in
-		/usr/lib|/usr/lib64|/lib|/lib64) ;;
-		*)
-		    output="$output -L$libdir"
-		    ;;
-	    esac
 	    output="$output @GPG_ERROR_CONFIG_LIBS@"
 	    ;;
         --host)
